### PR TITLE
Update EmailOTP URL parameters

### PR DESCRIPTION
**Pull request by asha15**
Related issue: https://github.com/wso2/product-is/issues/13779

### CODE REVIEW

**Review by ashensw — APPROVED** [review]

File: en/docs/learn/configuring-email-otp.md [hunk 1]
```diff
@@ -385,9 +385,9 @@ Follow the steps below to send the One Time Password (OTP) using Gmail APIs or u
     enable= true
 
     [authentication.authenticator.email_otp.parameters]
-    EMAILOTPAuthenticationEndpointURL = "https://localhost:9443/emailotpauthenticationendpoint/emailotp.jsp"
-    EmailOTPAuthenticationEndpointErrorPage = "https://localhost:9443/emailotpauthenticationendpoint/emailotpError.jsp"
-    EmailAddressRequestPage = "https://localhost:9443/emailotpauthenticationendpoint/emailAddress.jsp"
+    EMAILOTPAuthenticationEndpointURL = "https://localhost:9443/authenticationendpoint/email_otp.do"
```
[the remark this answers is not in the document]
Comment (asha15) [reply en/docs/learn/configuring-email-otp.md:388]: yes. I'll change in that raw as well

[the remark this answers is not in the document]
Comment (asha15) [reply en/docs/learn/configuring-email-otp.md:388]: @nilminiwso2 Updated the other endpoints.

**Comment by rksk** [comment]
This is incorrect! In 5.11.0, the email otp endpoint is a separate webapp.

**Comment by piraveena** [comment]
@asha15 pls revert this PR and send it to 6.0.0

**Pull request merged**